src/modules/producer.js

```diff
@@ -81,27 +109,28 @@ function publishOrSendToQueue(queue, msg, options) {
  * @param  {object} options contain rpc property (if true, enable rpc for this message)
  * @return {Promise}         Resolves when message is correctly sent, or when response is received when rpc is enabled
  */
-function checkRpc (queue, msg, options) {
+function checkRpc(queue, msg, options) {
   //messages are persistent
   options.persistent = true;
-
+  
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.dialonce.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)